client/src/components/ProfileDetail/ProfileDetail.tsx

```diff
@@ -0,0 +1,57 @@
+import { Avatar, Card, CircularProgress, Grid, Paper, Typography } from '@mui/material';
+import { useAuth } from '../../context/useAuthContext';
+import React from 'react';
+import { useStyles } from './useStyles';
+import { LocationOn } from '@mui/icons-material';
+
+const ProfileDetail = (): JSX.Element => {
```

Comment:
Please also create the request form component as per the mocks (the form used to make a service request)